tests/jenkins/install_neuron.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+export SPACK_INSTALL_PREFIX="${SPACK_INSTALL_PREFIX:-${WORKSPACE}/INSTALL_HOME}"
+BUILD_HOME="${WORKSPACE}/BUILD_HOME"
+export SPACK_ROOT="${BUILD_HOME}/spack"
+export PATH=$SPACK_ROOT/bin:/usr/bin:\$PATH
```

Comment:
Is this necessary? we typically `source spack/...setup_env.sh`.

Reply:
`export PATH=$SPACK_ROOT/bin:/usr/bin:\$PATH` is still needed. There are some issues with `git` when running `spack install` only by using `source spack/../setup_env.sh`